packages/app/src/debug/DebugResults.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div
+    class="flex gap-8px items-center"
+    data-cy="debug-results-holder"
+  >
+    <ResultCounts
+      :total-passed="results.totalPassed"
+      :total-failed="results.totalFailed"
+      :total-skipped="results.totalSkipped"
+      :total-pending="results.totalPending"
+      :order="['PASSED', 'FAILED', 'SKIPPED', 'PENDING']"
+    >
+      <template #prefix>
+        <div
+          class="text-red-400 font-semibold text-sm flex items-center border-r-1px px-2"
+        >
+          <component
+            :is="FailedSolidIcon"
+            class="h-16px w-16px pr-2"
+            data-cy="failed-prefix"
+          />
+          Failed
+        </div>
+      </template>
+    </ResultCounts>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { computed } from 'vue'
+import type { DebugResultsFragment } from '../generated/graphql'
+import { gql } from '@urql/core'
+import FailedSolidIcon from '~icons/cy/status-failed-solid_x24.svg'
+import ResultCounts from '@packages/frontend-shared/src/components/ResultCounts.vue'
```

Comment:
Curious what the team things but this seems like it's not really a good candidate for `frontend-shared`. It doesn't seem like something we will use in other packages (like launchpad).

Thoughts? cc @marktnoonan, we had a brief discussion around this.

Reply:
@lmiller1990 The `ResultsCounts` is an existing component that was just getting reused here with a minor addition.  I do agree that it is only used in `app` so does not need to be in `frontend-shared`, but we can refactor that another time.

Reply:
Sounds good to me